### ISSUE TITLE
name 'dataloaders' is not defined

### ISSUE THREAD

**Issue by aparnapanicker** [Shaam93/Building-a-classifer-with-Pytorch#1]
Hi , 

I have tried to execute the same code by using sampler to create validation set. As the dataloaders are separate, I am getting error when trying to execute the code as :name 'dataloaders' is not defined. May I know , what to do, inorder to combine the two loaders?

Thank you
